bitwarden_license/src/Sso/Utilities/Saml2OptionsExtensions.cs

```diff
@@ -81,13 +81,24 @@ public static async Task<bool> CouldHandleAsync(this Saml2Options options, strin
                 return false;
             }
 
-            if (assertion == null)
+            if (envelope == null)
             {
                 return false;
             }
 
+            if (options.SPOptions.WantAssertionsSigned)
```

Comment:
So this is great, however we need to know first if the `Issuer` is a match before we perform this test... otherwise in production (cloud) this will fail 99.9% of the time because the first profile that's tested/checked here will _not_ match (by signature or settings/configuration).